README.md

```diff
@@ -4,64 +4,66 @@ Example application using [adminjs](https://github.com/SoftwareBrothers/adminjs)
 
 ## Demo
 
-You can check out the current demo version at: https://admin-bro-example-app-staging.herokuapp.com/admin/login
+You can check out the current demo version at: https://adminjs-demo.herokuapp.com
```

Comment:
is different than `preview.adminjs.co`?

Reply:
preview.adminjs.co doesn't work yet (needs certificates)